src/homework4/decorators.py

```diff
@@ -0,0 +1,53 @@
+"""Создайте декоратор, который хранит в себе результаты вызовов функций и
+время получения результата за время запуска программы."""
+import time
+
+result = []
+
+
+def store(f):
+    def wrapper(*args, **kwargs):
+        start = time.time()
+        for i in range(1000):  # to make time bigger
+            res = f(*args, **kwargs)
+        stop = time.time()
+        result.append({"result": res % (10 ** 8), "time": (stop - start)})
+
+    return wrapper
+
+
+@store
+def pow(a, b):
+    return a ** b
+
+
+for i, j in zip(range(10 ** 2), range(10 ** 3)):
```

Comment:
can we use list comprehension [(i, j) for i in range for j in range] here? 
pow(*list_element)
zip is not usually used in python as all this functionality can be handled by comprehension or decorators.